Makefile

```diff
@@ -55,17 +68,24 @@ doc-preview: check_venv
 	@#sphinx-autobuild "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	sphinx-autobuild $(AUTOBUILD_OPTS) "docs/" "docs/_build/html/" $(SPHINXOPTS) $(O)
 
-doctest: check_venv
-	frost test -vv --doctest-modules --doctest-glob='*.py' -s --offline --debug-calls $(shell find . -type f -name '*.py' | grep -v venv | grep -v .pyenv | grep -v setup.py) \
-		--doctest-modules -s --offline --debug-calls
+# We need the list of all python file in several places, so only compute
+# it once.
+# TODO: move this to a Makefile variable
```

Comment:
Should we finish this TODO before merging?

Reply:
Or an enhancement issue if we want to remove all todo's.  It's a hygiene issue, not functional.

Reply:
removed TODO in next commit -- since the duplicate code is gone, there is no real need to re-implement this.